src/libraries/System.Reflection.MetadataLoadContext/tests/src/Tests/Module/ModuleTests.cs

```diff
@@ -233,5 +233,23 @@ public static void GetPEKindArm()
                 Assert.Equal(ProcessorArchitecture.Arm, pa);
             }
         }
+
+        [Fact]
+        public static void GetPEKindLoongArch64()
+        {
+            using (MetadataLoadContext lc = new MetadataLoadContext(new EmptyCoreMetadataAssemblyResolver()))
+            {
+                Assembly a = lc.LoadFromByteArray(TestData.s_PlatformLoongArch64);
```

Comment:
This test is testing ImageFileMachine.LOONGARCH64 that was reverted. It does not compile. Revert this change.

Reply:
OK, Thanks.